packages/calypso-build/package.json

```diff
@@ -22,10 +22,12 @@
 		"url": "https://github.com/Automattic/wp-calypso/issues"
 	},
 	"dependencies": {
+		"babel-loader": "8.0.5",
 		"css-loader": "2.1.1",
 		"mini-css-extract-plugin-with-rtl": "github:Automattic/mini-css-extract-plugin-with-rtl",
 		"postcss-loader": "3.0.0",
 		"sass-loader": "7.1.0",
+		"thread-loader": "2.1.2",
```

Comment:
Should these dependencies be removed from the root package.json?

Reply:
I think we tried and there was something else looking for it, going to try again.

Reply:
The server build is using it:

https://github.com/Automattic/wp-calypso/blob/cafed84500c748522bbd7461046ada893beb7469/webpack.config.node.js#L104-L116

Using the JSX build config would need to happen for the server as well.